docs/organizations/default-git-provider-integration.md

```diff
@@ -0,0 +1,18 @@
+# Configuring default settings for Git provider integration
+
+You can configure the default settings that Codacy uses to integrate with your Git provider when you add a new repository to Codacy. This enables you to apply the same settings across your organization repositories.
+
+To configure these default settings, go to the Git provider integration page for your organization. The integration features depend on your provider, [GitHub](../repositories-configure/integrations/github-integration.md#configuring), [GitLab](../repositories-configure/integrations/gitlab-integration.md#configuring) or [Bitbucket](../repositories-configure/integrations/bitbucket-integration.md#configuring).
+
+![Default Git provider integration](images/default-git-provider-settings.png)
+
+You can later change the integration settings for a specific repository on the repository **Settings**, tab **Integrations**.
+
+![Integration settings for repository](../repositories-configure/integrations/images/github-integration.png)
+
+## See also
+
+-   [Integrating Codacy with your Git workflow](../getting-started/integrating-codacy-with-your-git-workflow.md)
+-   [GitHub integration](../repositories-configure/integrations/github-integration.md)
+-   [GitLab integration](../repositories-configure/integrations/gitlab-integration.md)
+-   [Bitbucket integration](../repositories-configure/integrations/bitbucket-integration.md)
```

Comment:
Instead of repeating the navigation steps to the repository-specific Git provider integration settings on this page, what do you think of linking directly to the corresponding pages directly in the main section (instead of in the **See also** section)?

We already have links for each Git provider-specific page on the paragraph above, but I think it's possible to mention the links only once. Something like this:

> - The organization-level Git provider integration settings define the defaults that Codacy applies to new repositories
> - You can then customize the settings for each individual repository, which depend on your Git provider: [GitHub](https://docs.codacy.com/repositories-configure/integrations/github-integration/#enabling), [GitLab](https://docs.codacy.com/repositories-configure/integrations/gitlab-integration/#enabling), or [Bitbucket](https://docs.codacy.com/repositories-configure/integrations/bitbucket-integration/#enabling).

Reply:
Great suggestion! 👍 
I reorganized the content a bit to remove the redundant links.